src/docs/Components/Pagination/code.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Code'
+order: 2
+---
+
+## Import
+
+```scss
+@import 'settings-tools/_all-settings';
+@import 'components/_c.pagination';
+@import 'components/_c.select'; (Optionnal)
+```
+
+## Default Pagination
+
+
+```html
+<div class="mc-pagination example">
+  <a href="#" class="mc-pagination__button" aria-label="Pagination Navigation">
+    <svg class="mc-pagination__button-icon">
+      <use xlink:href="#iconLeftL" />
+    </svg>
+  </a>
+  <div class="mc-pagination__select">
+    <select class="mc-select mc-pagination__select-input">
+      <option value="">Page 1</option>
+    </select>
+  </div>
+
+  <a href="#" class="mc-pagination__button" aria-label="Pagination Navigation">
+    <svg class="mc-pagination__button-icon">
+      <use xlink:href="#iconRightL" />
+    </svg>
+  </a>
+</div>
+```
+
+<Preview path="pagination-default" />
+
+## Light Pagination 
+
+```html
+<div class="mc-pagination example">
```

Comment:
Can you remove the "example" class and put it on a separate div and add the **mc-pagination--light** modifier?

```HTML
<div class="example">
<div class="mc-pagination mc-pagination--light">...</div>
</div>
```